bastionzero/bastionzero.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 const (
-	libraryVersion = "v0.3.0"
+	libraryVersion = "v0.4.0"
```

Comment:
Don't need to increment. It is done automatically when we decide to release. All we need is a `changie` entry which you've made :)

Reply:
So yeah just remove this change

Reply:
c6aa5d0..726e06b